deepinv/models/diffunet.py

```diff
@@ -71,9 +71,14 @@ def __init__(
         use_new_attention_order = False
 
         out_channels = 6 if out_channels == 3 else out_channels
+        out_channels = 2 if out_channels == 1 else out_channels
```

Comment:
All the `if out_channels == 1` are a bit cumbersome. To add this to the architecture we would need to also upload the weights to HF and link it to a published paper. Would this be possible?